src/components/Editor.jsx

```diff
@@ -58,9 +62,18 @@ class Editor extends React.Component {
     window.removeEventListener('resize', this._handleWindowResize);
   }
 
-  _jumpToLine(line, column) {
-    this._editor.moveCursorTo(line, column);
+  _focusRequestedLine(requestedFocusedLine) {
+    if (get(requestedFocusedLine, 'language') !== this.props.language) {
+      return;
+    }
+
+    this._editor.moveCursorTo(
+      requestedFocusedLine.line,
+      requestedFocusedLine.column
+    );
+    this._editor.clearSelection();
```

Comment:
I threw in another fix for a related bug, which is that if your cursor is in a later position in the editor than the error you’re jumping to, the intervening text will be selected. Now it’s not.